docker/shadow-hand/indigo/intel_ide/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# Indigo with build tools Dockerfile
+#
+# https://github.com/shadow-robot/sr-build-tools/
+#
+
+FROM shadowrobot/build-tools:trusty-indigo-ide-intel
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$toolset_branch/bin/setup_dev_machine"
+
+RUN echo "Fixing file system" && \
+    mkdir -p /etc/apt/sources.list.d && \
+    rm -f /etc/apt/sources.list.d/ros-latest.list && \
+
+    echo "Running one-liner" && \
+    apt-get update && \
+    sudo apt-get -y install libgl1-mesa-glx libgl1-mesa-dri && \
```

Comment:
This should not be here, as it is already happening in the base image trusty-indigo-ide-intel.

This file should be exactly the same as `docker/shadow-hand/indigo/base/Dockerfile` except for the first `FROM` line.